### PR TITLE
News Panel

### PR DESCRIPTION
**Pull request by DarvishKamalia**

### CODE REVIEW

**Comment by DarvishKamalia** [comment]
I'm stupid pls ignore.

**Comment by thomassross** [comment]
Hi,

Just a couple of things I'd like to be changed:
1. News.cpp: instead of having three different `onFetchComplete<source>` functions, can you put it into one general one? They all do effectively the same thing, so this shouldn't be an issue.
2. Your code isn't fully in line with our [style guide](https://github.com/HorizonLauncher/Client/blob/dev/CONTRIBUTING.md) (Some braces are at the EOL, some extra newlines)
3.  Can you programmatically create the news widget instead of using a UI file?

Thanks.

**Comment by thomassross** [comment]
Just a couple of things:
1. Please revert the debug lines from CMakeLists.txt (line 6 & 16)
2. `newsitemwidget` should become `NewsItemWidget`
3. Please remove `Settings.ui`, as it has been removed here.
4. Please move `newspanel.ui` to a programmatically created UI
5. Please make sure that you are not using older versions of files (`MainPanel.cpp`)

File: Source/Settings.cpp [hunk 1]
```diff
@@ -446,3 +452,8 @@ void Settings::confirmClearDb()
             break;
     }
 }
+
+void Settings::manageNewsFeed() {
```
Comment (thomassross) [inline Source/Settings.cpp]: This can be made into a lambda

**Pull request merged**